sig-multicluster/hub-cluster-position-statement.md

```diff
@@ -0,0 +1,57 @@
+# Hub Cluster - SIG Multicluster Position Statement
+
+Author: Corentin Debains (**[@corentone](https://github.com/corentone)**), Google  
+Last Edit: 2025/01/25  
+Status: DRAFT
+
+## Goal
+To establish a standard definition for a central cluster that is leveraged by multicluster 
+controllers to manage multicluster applications or features across an inventory of clusters.
+
+## Context
+Multicluster controllers have always needed a place to run. This may happen in external
+proprietary control-planes but for more generic platforms, it has been natural for the
+Kubernetes community to leverage a Kubernetes Cluster and the existing api-machinery
+available. There has been a variety of examples of which we can quote ArgoCD, MultiKueue
+or any of the Federation effort (Karmada, KubeAdmiral), all of them not-naming the "location"
+where they run or not aligning on the name (Admin cluster, Management Cluster, Command Cluster, Manager Cluster...). 
+The [ClusterInventory](https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/4322-cluster-inventory/README.md)
+(ClusterProfile CRDs) is also the starting point for a lot of multicluster controllers and, 
+being a CRD, it requires an api-machinery to host it. The functionality of this cluster is also
+defined in separation to what a "workload" cluster does, which is to run the business applications,
+when hub runs infrastructure components.
+
+## Definition
+
+A (multicluster) hub cluster is a Kubernetes cluster that acts as a 
+control-plane for other Kubernetes clusters (named Workload Clusters to differentiate
+them). It MUST have the ClusterProfiles written on it MAY have access to the api, metrics or
```

Comment:
Is this definition intended to cover what Open Cluster Management does? OCM has its own representation of a workload (execution) cluster, called `ManagedCluster`.

Reply:
the goal here is to bind it with ClusterProfile. while we can draw the parallel with OCM, I think both this and ClusterProfile gain at being linked together.

Reply:
Great! We have another option.